assets/models/system/phi-3-mini-128k-instruct/description.md

```diff
@@ -0,0 +1,190 @@
+|WinoGrande <br> 5-Shot |72.50|82.50|81.40|54.7|54.20|55.60|62.00|68.80|
+|OpenBookQA <br> 10-Shot |80.60|88.40|87.20|73.6|79.80|78.60|85.80|86.00|
+|BoolQ <br> 0-Shot |78.70|82.90|86.60| -- |72.20|66.00|76.60|79.10|
+|CommonSenseQA <br> 10-Shot |78.00|80.30|82.60|69.3|72.60|76.20|78.10|79.60|
+|TruthfulQA <br> 10-Shot |63.20|68.70|75.70| -- |53.00|52.10|60.10|67.70|
+|HumanEval <br> 0-Shot |57.90|59.10|55.50|59|28.00|34.10|37.80|62.20|
+|MBPP <br> 3-Shot |62.50|71.40|74.50|60.6|50.80|51.50|60.20|77.80|
+
+## How to Use
+
+### **Deployment**
+To deploy the model simply click on the "Deploy" button on the toolbar of this page. Detailed instructions can be found [here](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-use-foundation-models?view=azureml-api-2#deploying-foundation-models-to-endpoints-for-inferencing). 
+
+### **Sample inputs and outputs (for real-time inference)**
+Once deployment is successful, navigate to the deployed endpoint page. On that page you will find "Consume" tab. There you will find sample code in JavaScript, Python, C# and R.
+
+
+#### **Sample input**
+```json
+{
+  "input_data": {
+    "input_string": [
+      {
+        "role": "user",
+        "content": "I am going to Paris, give me a list of 10 places to visit"
+      }
+    ],
+    "parameters": {
+      "temperature": 0.7,
+      "top_p": 0.9,
+      "do_sample": true,
+      "max_new_tokens": 1000
+    }
+  }
+}
+``` 
+#### **Sample output**
+```json
+{
+  "output": " 1. Eiffel Tower: Visit the iconic symbol of Paris, offering breathtaking views of the city.\n\n2. Louvre Museum: Explore one of the world's largest and most visited museums, home to thousands of works of art, including the Mona Lisa.\n\n3. Notre-Dame Cathedral: Marvel at the stunning Gothic architecture of this famous cathedral, although note that it is currently under renovation due to the 2019 fire.\n\n4. Montmartre: Discover this historic and artistic neighborhood, famous for its bohemian past and the stunning Sacré-Cœur Basilica.\n\n5. Seine River Cruise: Take a relaxing cruise on the Seine River, seeing some of the city's most famous landmarks like the Louvre, Notre-Dame, and the Eiffel Tower from a unique perspective.\n\n6. Champs-Élysées: Visit this famous avenue lined with shops, cafes, and theaters. Don't forget to check out the Arc de Triomphe at its end.\n\n7. Palace of Versailles: Take a day trip from Paris to explore the opulent palace and gardens of Versailles, a UNESCO World Heritage site.\n\n8. Sacré-Cœur Basilica: Located at the highest point in the city, this basilica offers panoramic views of Paris.\n\n9. Latin Quarter: Stroll through this historic and vibrant neighborhood, famous for its student life, lively atmosphere, and cafes.\n\n10. Musée d'Orsay: Visit this museum, housing an impressive collection of Impressionist and Post-Impressionist art, including works by Monet, Degas, Renoir, and Van Gogh."
+}
+```
+
```

Comment:
Inference / Finetuning samples subsequently together as H2 and tabular format (please refer to other model cards for reference)